src/Mount.cpp

```diff
@@ -2115,6 +2087,13 @@ void Mount::loop() {
             startSlewing(TRACKING);					   
           }
         #endif
+
+        // Reset DEC to guide microstepping so that guiding is always ready and no switch is neccessary on guide pulses.
+        #if DEC_STEPPER_TYPE == STEPPER_TYPE_NEMA17 && DEC_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART
```

Comment:
do we really need to check for the stepper type here? TMC2209 is being used with nemas only

Reply:
and even if its something else ... the actual logic is based on the driver only

Reply:
True

Reply:
Fixed